app/assets/javascripts/components/my_lessons_page.es6.jsx

```diff
@@ -0,0 +1,62 @@
+class MyLessonsPage extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      filter: 'upcoming',
+    }
+  }
+
+  handleClick(filter) {
+   this.setState({filter: filter});
```

Comment:
can you add a space after the braces? so like `this.setState({ filter: filter });` just for style purposes!